src/components/notes/List/NoteRow.jsx

```diff
@@ -10,6 +10,11 @@ import Alerter from 'cozy-ui/transpiled/react/Alerter'
 import ActionMenu, { ActionMenuItem } from 'cozy-ui/transpiled/react/ActionMenu'
 import { TableRow, TableCell } from 'cozy-ui/transpiled/react/Table'
 import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
+import FilePath from 'cozy-ui/transpiled/react/FilePath'
+import AppLinker from 'cozy-ui/transpiled/react/AppLinker'
+import { Link } from '@material-ui/core'
```

Comment:
from cozy-ui. You should not import from mui directly.

Reply:
yes it's just because it doesn't work in yarn link mode (these types of big blunders could be frequent in draft mode)

Reply:
Even if it's a draft or else, you should have something working with the correct configuration. If yarn link doesn't work, you can use rlink (https://gist.github.com/ptbrowne/add609bdcf4396d32072acc4674fff23)

Reply:
Will do